release-notes/aws/v11.3.3.md

```diff
@@ -0,0 +1,10 @@
+## :zap: Giant Swarm Release v11.3.3 for AWS :zap:
+
+**If you are upgrading from 11.3.2, upgrading to this release will not roll your nodes. It will only update the apps.**
+
+This release fixes an issue where `cert-manager` could be killed for exceeding its memory limit.
```

Comment:
@glitchcrab thanks for drafting! could you add some implication to customer, how does it negatively affect them when cert manager is killed?

Reply:
done - english is hard :see_no_evil: